dev/com.ibm.ws.transport.iiop/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -23,6 +23,8 @@ orbWrapper.iiopEndpoint.desc=Optional IIOP Endpoint describing the ports open fo
 orbWrapper.name=Object Request Broker (ORB)
 orbWrapper.nameService=Naming context location for a client ORB
 orbWrapper.nameService.desc=Optional URL for the remote name service, for example corbaname::localhost:2809
+orbWrapper.orbSSLInitTimeout=Timeout that governs ORB SSL initialization
```

Comment:
I noticed that these descriptions have no periods. If that is the convention for the other labels, then OK. Otherwise, let's make them complete sentences.